diff --git a/node_modules/postgres/cjs/src/types.js b/node_modules/postgres/cjs/src/types.js
index 0578284..40cc48e 100644
--- a/node_modules/postgres/cjs/src/types.js
+++ b/node_modules/postgres/cjs/src/types.js
@@ -29,7 +29,7 @@ const types = module.exports.types = {
     to: 1184,
     from: [1082, 1114, 1184],
     serialize: x => (x instanceof Date ? x : new Date(x)).toISOString(),
-    parse: x => new Date(x)
+    parse: x => x /* patched: return string not Date */
   },
   bytea: {
     to: 17,
diff --git a/node_modules/postgres/src/types.js b/node_modules/postgres/src/types.js
index 7c7c2b9..cebcfcd 100644
--- a/node_modules/postgres/src/types.js
+++ b/node_modules/postgres/src/types.js
@@ -29,7 +29,7 @@ export const types = {
     to: 1184,
     from: [1082, 1114, 1184],
     serialize: x => (x instanceof Date ? x : new Date(x)).toISOString(),
-    parse: x => new Date(x)
+    parse: x => x /* patched: return string not Date */
   },
   bytea: {
     to: 17,
